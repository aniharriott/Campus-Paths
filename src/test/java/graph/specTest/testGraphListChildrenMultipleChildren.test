# Test Graph listChildren multiple children file.
# Tests that given parent node with multiple
# children, all its children will be printed.

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add Edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2

# Print children of n1
ListChildren graph1 n1