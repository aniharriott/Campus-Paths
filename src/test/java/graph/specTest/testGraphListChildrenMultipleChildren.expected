# Test Graph listChildren multiple children file.
# Tests that given parent node with multiple
# children, all its children will be printed.

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add Edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1

# Print children of n1
the children of n1 in graph1 are: n2(e1) n3(e2)